site/docs/v1/tech/lambdas/epic-games-reconcile.adoc

```diff
@@ -0,0 +1,65 @@
+---
+layout: doc
+title: Epic Games Reconcile
+description: An overview of the Epic Games Reconcile lambda
+---
+
+:sectnumlevels: 0
+
+== Epic Games Reconcile lambda
+
+When an Epic Games identity provider is used to complete a federated login request FusionAuth will use the email address to reconcile user. You may optionally utilize a lambda to customize the user and user registration during this authentication event.
```

Comment:
I'm not 100%, but I think with the new `linkingStrategy` approach "FusionAuth will use the email address or username to reconcile **the** user".  In fact, it's really more like email, username, `identityProviderUserId`, or nothing (`LinkAnonymously`?) depending on the `linkingStrategy`.

Reply:
Each IdP has a default linking strategy, but with the exception of HYPR (I think), the strategy can be modified. So if we are going to describe a linking strategy in the overview here, it will need to be more generic as it is not set in stone. 

Epic Games will default to `CreatePendingLink`, that should be documented in the API, and possibly if there is a UI workflow overview somewhere. 

The lambda will not be used when the strategy is set to anonymous. 

This goes for all of these new Lambda docs.